### PR TITLE
docs: add more info about isInvalid props

### LINKED ISSUE #5064: Switch component does not apply an `isInvalid` style
### Description

When the `isInvalid` prop is applied to the `Switch` component. I would expect some visual representation of this.

### Link to Reproduction

https://codesandbox.io/s/chakra-switch-isinvalid-efj7b?file=/src/index.tsx

### Steps to reproduce

1. Go to `index.tsx`
2. See that the component with `isInvalid` has no styling applied

### Chakra UI Version

1.6.12

### Browser

Google Chrome 95

### Operating System

- [X] macOS
- [ ] Windows
- [ ] Linux

### Additional Information

The [docs](https://chakra-ui.com/docs/form/switch#props) state that it updates the state of the unchecked state. Not sure if the component or the documentation needs to change.

### PR DESCRIPTION
**Pull request by amustaque97**
Closes #5064 

## 📝 Description
Docs has been updated to explicitly mention that `isInvalid` props will not change any style.
> Add a brief description

## ⛳️ Current behavior (updates)
In the issue reported, User is expecting some visual change but that's not happening. 
> Please describe the current behavior that you are modifying

## 🚀 New behavior
New docs will make it clear to the user that there will be no change in style in case of `Switch` component.
> Please describe the behavior or changes this PR adds

## 📝 Additional Information
**Screenshot**:
After making changes
image

### CODE REVIEW

**Review by sohamsshah — APPROVED** [review]
LGTM! Thanks for the docs update! 🎉

**Comment by noobinthisgame** [comment]
@amustaque97 Thanks for this PR. As the docs have been moved to their own repo before we managed to merge this it would be nice if you could reopen a PR in the [docs repo](https://github.com/chakra-ui/chakra-ui-docs). :)

**Comment by noobinthisgame** [comment]
The changes here can then be merged aswell :)

**Comment by amustaque97** [comment]
> The changes here can then be merged aswell :)

@noobinthisgame, if this issue is still present then I can raise a PR there! Since it has been more than 2 months, I'm not sure if issue is still there or not!

**Comment by noobinthisgame** [comment]
@amustaque97 the issue is still open but has been transfered to the docs repo :)

**Comment by TimKolberger** [comment]
Kindly mention this in the docs on the Switch component page. 
Closing this because it is a documentation issue for `Switch`, not `useCheckbox`.

**Pull request closed without merging**
